### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by krolich500grn**
- [DEMO LINK](https://krolich500grn.github.io/react_list-of-goods-hooks/)

### CODE REVIEW

File: src/components/GoodsList/GoodsList.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+type Props = {
+  goods: string[],
+};
+
+export const GoodsList: React.FC<Props> = React.memo(
+  ({ goods }) => (
+    <ul className="Goods__list">
+      {goods.map(good => (
+        <li className="Goods__item" key={good}>
```
Comment (aspurgilus) [inline src/components/GoodsList/GoodsList.tsx:13]: It's better to use uuid that you have already installed.

File: src/components/GoodsList/GoodsList.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import './GoodsList.scss';
+
+type Props = {
+  goods: string[],
+};
+
+export const GoodsList: React.FC<Props> = React.memo(
```
Comment (aspurgilus) [inline src/components/GoodsList/GoodsList.tsx:9]: Using memo seems to be a good solution but all you can do in parent component is pushing the start button to render the child.

**Review by aspurgilus — APPROVED** [review]
Good work!

File: src/App.tsx [hunk 3]
```diff
@@ -12,13 +18,93 @@ const goodsFromServer: string[] = [
...
+            </button>
+          </div>
+        )
+        : (
+          <>
+            <div className="App__sort-by">
+              <button
+                type="button"
+                className="button"
+                onClick={() => reverse()}
```
Comment (solaryasha) [inline src/App.tsx:73]:
```suggestion
                onClick={reverse}
```

**Pull request closed without merging**